helm/cluster-cloud-director/values.schema.json

```diff
@@ -441,6 +500,26 @@
                 "highest"
             ]
         },
+        "registry": {
+            "type": "object",
+            "properties": {
+                "configure": {
+                    "type": "boolean"
+                },
```

Comment:
Would be great to have a title here, and a description saying what activating this will do.